packages/victory-brush-container/src/brush-helpers.js

```diff
@@ -266,7 +281,11 @@ const Helpers = {
     const parentSVG = targetProps.parentSVG || Selection.getParentSVG(evt);
     const { x, y } = Selection.getSVGEventCoordinates(evt, parentSVG);
     // Ignore events that occur outside of the maximum domain region
-    if ((!allowResize && !allowDrag) || !this.withinBounds({ x, y }, fullDomainBox)) {
+    if (
+      (!allowResize && !allowDrag) ||
+      !this.withinBounds({ x, y }, fullDomainBox) ||
+      !this.hasMoved({ ...targetProps, x2: x, y2: y })
```

Comment:
There is currently a check for whether the mouse moved in `onMouseUp`. I think the check should stay there, right?

Reply:
I tried to do it there initialy, but the drawing happens even before the mouseup

Reply:
Oh, I see what you mean. Since this has to run on every check, it would be great to be able to short circuit it for users who don't have a `mouseMoveThreshold` defined. Like `(targetProps.mouseMoveThreshold && !this.hasMoved({ ...targetProps, x2: x, y2: y }))`

It might also be nice to reuse this logic in `onMouseUp` instead of the more naive check

Reply:
> reuse this logic in `onMouseUp`

Why? `x2` won't be different from `x1` unless `onMouseMove` handling actually happens

Reply:
General preference for a named function like `hasMoved`, but I guess `(x1 === x2 || y1 === y2)` is already pretty concise and clear.